src/pipelines/epidemiology/config.yaml

```
@@ -109,6 +109,19 @@ sources:
       job_group: all
       deferred: true
 
+  # Data sources for AT level 3
+  - name: pipelines.epidemiology.at_authority.AustriaLevel3DataSource
+    fetch:
+      - url: "https://covid19-dashboard.ages.at/data/CovidFaelle_Timeline_GKZ.csv"
+    parse:
+      country: AT
+      sep: ";"
+    test:
+      location_key_match: 'AT_.+'
```

Tests probably pass because there's at least one district picked at random from the locations starting with `AT_*`, but to be more precise this should instead be:

```suggestion
      location_key_match: 'AT_[^_]+[^_]+'
```

That way, we can run `update.py --only epidemiology --location-key AT_*_*` and only this data source will run.

---

After making this change, I can't get the data source to run with "update.py --only epidemiology --location-key AT_*_*" or with "update.py --only epidemiology --location-key AT_*" -- in both cases, only the level 2 Austria source runs and not the level 3.